ci/run_clang_tidy.sh

```diff
@@ -46,7 +46,7 @@ function filter_excludes() {
 
 if [[ "${RUN_FULL_CLANG_TIDY}" == 1 ]]; then
   echo "Running full clang-tidy..."
-  run-clang-tidy-8
+  run-clang-tidy-9
 elif [[ -z "${CIRCLE_PR_NUMBER}" && "$CIRCLE_BRANCH" == "master" ]]; then
   echo "On master branch, running clang-tidy-diff against previous commit..."
   git diff HEAD^ | filter_excludes | clang-tidy-diff-8.py -p 1
```

Comment:
`clang-tidy-diff-9.py`?

(and also line 57)

Reply:
nice catch, thanks